.github/workflows/tests_gpu_python.yml

```
@@ -183,7 +183,7 @@ jobs:
         run: |
           rm -rf build
           PL_BACKEND=lightning_qubit python scripts/configure_pyproject_toml.py || true
-          PL_BACKEND=lightning_qubit SKIP_COMPILATION=True python -m pip install . -vv
+          PL_BACKEND=lightning_qubit python -m pip install . -vv
```

Could we skip the LQ compilation?
```suggestion
          PL_BACKEND=lightning_qubit SKIP_COMPILATION=True python -m pip install . -vv
```

---

Let's revert this change but modify the unit tests.